R/annotation.R

```diff
@@ -80,7 +81,37 @@ annotate_ssm_blacklist = function(mutations_df,
   mutations_df = left_join(mutations_df,combined_blacklist,by = c("Chromosome", "Start_Position")) %>%
     mutate(blacklist_count = replace_na(blacklist_count, 0))
 
+  if (use_curated_blacklist){
+    repo_base = config::get("repo_base")
+    full_path = paste0(repo_base, config::get("results_versioned")$curated_blacklist)
+    additional_blacklist = glue(repo_path) %>% read_tsv(c("chrpos", "blacklist_count2"), show_col_types = FALSE)
```

Comment:
I think it's better to have a header in your file so that we don't need to hardcode this in GAMBLR